proof_of_stake/src/parameters.rs

```diff
@@ -46,6 +46,9 @@ pub struct PosParams {
     /// The minimum amount of bonded tokens that a validator needs to be in
     /// either the `consensus` or `below_capacity` validator sets
     pub validator_stake_threshold: token::Amount,
+    /// The number of blocks in the past for which the first block height of an
+    /// epoch is recorded
+    pub max_age_num_blocks: u64,
```

Comment:
If we were to use this param and pass it to CometBFT, it would invalidate any evidence that's more than this number of blocks in the past, so we'd need to ensure that it's greater than our slashable number of epochs * `min_num_of_blocks`, but that's complicated because the number of blocks in epoch may vary, so I think we're better off not using this param and filter out outdated evidence in the app (as we already do)